src/libsyncengine/syncpal/operationprocessor.cpp

```diff
@@ -32,9 +32,10 @@ bool OperationProcessor::isPseudoConflict(std::shared_ptr<Node> node, std::share
     }
 
     std::shared_ptr<Snapshot> snapshot =
-        (node->side() == ReplicaSide::ReplicaSideLocal ? _syncPal->_localSnapshot : _syncPal->_remoteSnapshot);
+        (node->side() == ReplicaSide::ReplicaSideLocal ? _syncPal->_localSnapshotCopy : _syncPal->_remoteSnapshotCopy);
```

Comment:
Suggestion: create a helper function that retrieves the snapshot copy given the replica side as input (the same operation is done several times below).

Reply:
Done